laborator/content/buffer-overflow/solution/9-return-address/Readme.md

```diff
@@ -0,0 +1,14 @@
+# Writeup
+
+- First use this command to scan the exec:
+> ```Bash
+> objdump -M intel -d break_this
+> ```
+- `magic_function()` is saved at `0x08048596`
+- From the first 4 lines from `read_buffer()` we get that: `ebp` and `edi` get pushed on stack then the stack is extended by `0x54` = `84`
+- So we must print `88 + 4 + 4 = 92` dummy characters `A` and then the address of `magic_function()` in little-endian encoding
+- I will set `n` to a big enough value: `128`
```

Comment:
You don't specify what this `n` is exactly. I think you should break up this writeup into 2 sections (within the same file). The first one should explain what the binary does (based on snippets from `objdump`'s output) and the latter should explain the exploit.